.github/workflows/build.yml

```diff
@@ -14,7 +14,10 @@ jobs:
     name: Build
     strategy:
       matrix:
-        python-version: [ '3.9', '3.10', 'pypy3.9' ]
+        python-version:
+          - ${{ vars.ARCALOT_PYTHON_VERSION }}
+          - '3.10'
+          - 'pypy3.9'
```

Comment:
This is a little weird in that we've got a variable for the "default/main" version, but hardcoded references for other testing platforms. E.g., if we change `ARCALOT_PYTHON_VERSION` to 3.10, we'll be testing that twice unless we manually update this workflow (in which case there's no real advantage in using the variable in the first place).

Reply:
This is true.  However, I think this entire workflow should be rototilled -- the "publish" portion should use the [`python_release`](https://github.com/arcalot/arcaflow-reusable-workflows/blob/main/.github/workflows/python_release.yaml) reusable workflow, and the lint and test portion should be refactored into an analog of [`go_lint_and_test`](go_lint_and_test) which should address the matrix -- so I'll defer addressing that to another PR.

Reply:
The advantage of using the variable is that it ensures that at least one iteration is guaranteed to match for producing the artifact for the `publish` job to publish....